documentation/specs/proposed/RAR_caching_onepager.md

```diff
@@ -0,0 +1,60 @@
+## RAR caching
+RAR in this context means resolving of assembly references.
```

Comment:
This is a nit: if you want to make this a bit less wordy:
"Resolving of Assembly References (RAR)" and then continue using RAR after that.